### DISCUSSION TITLE
Tiny Stickers for the Controller Buttons and Triggers

### DISCUSSION THREAD

**Discussion by SecularSteve** [inputlabs/organization#64] [Ideas & feature request]
It can either be a Supporter Exclusive, or something people can order from you as a thank you.

I already have an idea what they can look like if there's anything to gain from this.

**Comment by marcos-diaz** [comment]
Sorry for the late reply, I missed this. 

Some people in the Discord server shared different ways to customize the buttons with different glyphs or colors. Stickers is also a good idea, but I would say it goes into the category of community mods rather than in the reference designs.
